Containerfile

```diff
@@ -16,32 +16,32 @@ RUN dnf -yq install \
 
 ADD . /pulp-fixtures
 
-RUN make -C pulp-fixtures all-fedora base_url=http://BASE_URL
+RUN make -C pulp-fixtures fixtures/python-pypi base_url=http://BASE_URL
 
 # === Build fixtures (Debian) =================================================
-FROM debian:stretch AS debian-build
-
-RUN apt-get update
-RUN apt-get -y install \
-              equivs \
-              reprepro \
-              dpkg-dev \
-              apt-utils
-
-RUN mkdir -p /pulp-fixtures/fixtures
-ADD Makefile /pulp-fixtures
-ADD common /pulp-fixtures/common
-ADD debian /pulp-fixtures/debian
-
-RUN make -C pulp-fixtures all-debian
+# FROM debian:stretch AS debian-build
+#
+# RUN apt-get update
+# RUN apt-get -y install \
+#               equivs \
+#               reprepro \
+#               dpkg-dev \
+#               apt-utils
+#
+# RUN mkdir -p /pulp-fixtures/fixtures
+# ADD Makefile /pulp-fixtures
+# ADD common /pulp-fixtures/common
+# ADD debian /pulp-fixtures/debian
+#
+# RUN make -C pulp-fixtures all-debian
 
 # === Serve content ===========================================================
 FROM registry.access.redhat.com/ubi8/ubi AS server
 
 RUN dnf install -y nginx && dnf clean all
 RUN rm /usr/share/nginx/html/index.html
 COPY --from=fedora-build pulp-fixtures/fixtures /usr/share/nginx/html
-COPY --from=debian-build pulp-fixtures/fixtures /usr/share/nginx/html
+# COPY --from=debian-build pulp-fixtures/fixtures /usr/share/nginx/html
```

Comment:
Is this PR maybe still draft?

Reply:
Woops, accidentally added my modified containerfile.